app/src/main/java/org/scesi/cappuchino/ui/utils/SearchBar.kt

```diff
@@ -0,0 +1,157 @@
+            )
+        },
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(dimensionResource(id = R.dimen.margin_padding_size_xxxlarge))
+            .padding(horizontal = dimensionResource(id = R.dimen.margin_padding_size_smallv1))
+            .clip(RoundedCornerShape(dimensionResource(id = R.dimen.margin_padding_size_small)))
+            .onFocusEvent { focusState ->
+                if (focusState.isFocused) {
+                    isSearchBarFocused(false)
+                } else {
+                    isSearchBarFocused(true)
+                }
+            },
+            colors = TextFieldDefaults.outlinedTextFieldColors(
+            focusedBorderColor = Color.Gray,
+            unfocusedBorderColor = Color.Gray
+        ),
+        shape = RoundedCornerShape(dimensionResource(id = R.dimen.margin_padding_size_small))
+    )
+    Spacer(modifier = Modifier.height(16.dp))
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .verticalScroll(rememberScrollState())
+    ) {
+        if (filteredCarreras.isEmpty()) {
+            Text(
+                text = stringResource(id = R.string.not_found),
+                color = Color.Gray,
+                modifier = Modifier.align(Alignment.CenterHorizontally)
+            )
+        } else {
+            filteredCarreras.forEach { carrera ->
+                TextBoxSearch(text = carrera.searchCriteria, onClick = {
+                })
+            }
+        }
+    }
+        }
+}
+
+
+@Composable
+fun TextBoxSearch(
+    text: String,
+    onClick: () -> Unit
+) {
+
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .wrapContentHeight()
+            .padding(dimensionResource(id = R.dimen.margin_padding_size_smallv1))
+            .background(
+                color = Color(0xFFD1D8FF),
+                shape = RoundedCornerShape(dimensionResource(id = R.dimen.margin_padding_size_small))
+            )
+            .clickable { onClick() }
+            .padding(
+                horizontal = dimensionResource(id = R.dimen.margin_padding_size_small),
+                vertical = dimensionResource(id = R.dimen.margin_padding_size_smallv3)
+            ),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        Text(
+            text = text,
+            fontSize = dimensionResource(id = R.dimen.text_size_small).value.sp,
+            color = Color.Black
+        )
+        Icon(
+            painter = painterResource(id = R.drawable.fecha),
+            contentDescription = stringResource(R.string.arrow_icon),
+            tint = Color.Black,
+            modifier = Modifier.size(dimensionResource(id = R.dimen.margin_padding_size_medium))
+        )
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+fun PreviewSearchBar() {
+    SearchBar(career = SearchCategory.subjectsExample,isSearchBarFocused = { isFocused -> })
```

Comment:
not sure if you are using the plugin , but since you specify the definition of isSearchBarFocuse i guess not 
![image](https://github.com/user-attachments/assets/7c0ee870-a396-4650-b033-c2f3daed783c)
and only replace the list of career